packages/api/src/index.ts

```diff
@@ -38,24 +38,25 @@ async function main () {
 function readDataFeeds (): Array<FeedInfo> {
   console.log('DIRNAME', __dirname)
   console.log('DATA_FEED_CONFIG_PATH:>', process.env.DATA_FEED_CONFIG_PATH)
-  console.log('resolve path: ', path.resolve(process.env.DATA_FEED_CONFIG_PATH || './dataFeeds.json'))
+  console.log(
+    'resolve path: ',
+    path.resolve(process.env.DATA_FEED_CONFIG_PATH || './dataFeeds.json')
+  )
```

Comment:
Preferently, let's move `path.resolve(process.env.DATA_FEED_CONFIG_PATH || './dataFeeds.json')` into a `const` statement before the logging, then we can reuse on line `48` to avoid code repetition and ensure that we are logging the path that is actually being used.